de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/InstantIncomingReferencePointNegotiation.java

```diff
@@ -59,24 +59,25 @@ public InstantIncomingProjectNegotiation(
 
     @Override
     protected void transfer(IProgressMonitor monitor,
-        Map<String, IProject> projectMapping, List<FileList> missingFiles)
-        throws IOException, SarosCancellationException {
+        Map<String, IReferencePoint> referencePointMapping,
+        List<FileList> missingFiles) throws IOException,
+        SarosCancellationException {
 
         awaitActivityQueueingActivation(monitor);
 
         /*
          * the user who sends this ProjectNegotiation is now responsible for the
          * resources of the contained projects
          */
-        for (Entry<String, IProject> entry : projectMapping.entrySet()) {
-            final String projectID = entry.getKey();
-            final IProject project = entry.getValue();
+        for (Entry<String, IReferencePoint> entry : referencePointMapping
+            .entrySet()) {
+            final String referencePointID = entry.getKey();
+            final IReferencePoint referencePoint = entry.getValue();
 
-            referencePointManager.put(project.getReferencePoint(), project);
-            session.addReferencePointMapping(projectID,
-                project.getReferencePoint());
+            // referencePointManager.put(project.getReferencePoint(), project);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWc2FteW0EcSutaxNIIf&open=AWc2FteW0EcSutaxNIIf)

[ISSUE_KEY]: AWc2FteW0EcSutaxNIIf